MIDAS/src/hardware/GPSSensor.cpp

```diff
@@ -17,17 +17,53 @@ ErrorCode GPSSensor::init() {
     return ErrorCode::NoError;
 }
 
+// This is needed because GPS doesn't provide unix time and just gives
+// dd mm yy
+uint16_t months[12] = {
+    0, 31, 59, 90, 120, 151, 181, 212, 243, 273, 304, 334
+};
+
+inline bool is_leapyear(int year) {
+    return (!(year % 100 == 0) || (year % 400 == 0)) && year % 4 == 0;
```

Comment:
`!(year % 100 == 0)` is sometimes called `year % 100 != 0`

Reply:
I think !(year % 100) also works